test/module/irohad/multi_sig_transactions/gossip_propagation_strategy_test.cpp

```diff
@@ -1,35 +1,23 @@
 /**
- * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
- * http://soramitsu.co.jp
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *        http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
  */
 
 #include "multi_sig_transactions/gossip_propagation_strategy.hpp"
-#include <gmock/gmock.h>
-#include <gtest/gtest.h>
+
 #include <algorithm>
-#include <boost/range/irange.hpp>
 #include <iterator>
 #include <memory>
 #include <numeric>
-#include <rxcpp/rx.hpp>
 #include <string>
 #include <thread>
 #include <vector>
 
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <boost/range/irange.hpp>
```

Comment:
shouldn't it be higher?

Reply:
No, since these are not system libraries.